test/language/statements/class/static-init-invalid-await.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-class-definitions
+description: The "Await" parsing context does not apply to the block's statement list
+info: |
+  Syntax
+
+  [...]
+
+  ClassStaticBlockStatementList :
+     StatementList[~Yield, ~Await, ~Return]opt
```

Comment:
Per https://github.com/tc39/proposal-class-static-block/pull/46, this changes to:
```suggestion
     StatementList[~Yield, +Await, ~Return]opt
```
However _AwaitExpression_ and `for await` are both early errors in static semantics.